packages/nimble-components/src/wafer-map/specs/README.md

```diff
@@ -63,6 +63,8 @@ Please see below a few screenshots regarding the current `wafer-map` component:
 From a user’s perspective one of the most important actions which this component should be able to achieve is to zoom in and out on the datapoints. This functionality should occur when the mouse pointer hovers the `nimble-wafer-map` component and a mouse wheel action is performed.
```

Comment:
@munteannatan I think as-is the api changes proposed are not controversial and the implementation sounds well-aligned with current architecture so we can go ahead and merge this as @jattasNI is OOO. If Jesse has additional feedback we can discuss it in a follow-up.